src/skeleplex/graph/skeleton_graph.py

```diff
@@ -407,3 +415,95 @@ def compute_branch_lengths(self) -> dict:
 
         nx.set_edge_attributes(self.graph, edge_lengths, LENGTH_KEY)
         return edge_lengths
+
+    def sample_volume_slices_from_spline(
```

Comment:
I would prefer if we split this up into two functions: 
- a function that takes the spline and volume, etc. and returns a numpy array of the slices
- a function that takes the numpy array and writes the h5 file.

I think this abstraction is useful because it allows us to re-use the slicing part in other parts of the code base (e.g., in the viewer)

Reply:
Mh, so I could return a dictionary of dictionaries which contains then the array. Eg.

orthogonal_edge_slice_dict['image']['(123,456)']

And then I create a utils package where I have a function that uses that dict as an input to write h5 containers?
A function for a single spline is already in the spline object, so I dont know if we need it in the SkeletonGraph, too.